### DISCUSSION TITLE
post/more-on-ipv6-firewall/

### DISCUSSION THREAD

**Discussion by giscus** [ecwu/ecwu.github.io#6] [Blog Post Discussion]
# post/more-on-ipv6-firewall/

Homepage and Blog of ECWU

https://ecwuuuuu.com/post/more-on-ipv6-firewall/

**Comment by weeeo** [comment]
do you have any lucky to solve the lnksys ipv6 firewall setting problem?
I have the same problem.
after contacting, I got no solution.

**Reply by ecwu** [reply]
Unfortunate to tell you: but their tech support has never reached out to me since then. :(

So my current solutions are:
- Open the IPv6 firewall without any rule applied.
- Use FRP to expose the service I need when I am outside of my home.

**Reply by weeeo** [reply]
my current solution is running a self-developed python program to renew firewall rule automatically.

**Reply by ecwu** [reply]
That sounds like a good solution. Can you share more details? Are you using python to simulate the browser action and submit the request? Or using UPnP to edit router configuration?

**Reply by weeeo** [reply]
I have old laptops runing ubuntu as small home servers.  And I am using selenium to simulate browser. If you are considering this simulation, you need to know the headless mode is buggy for both Firefox and Chrome.
